scipy/optimize/_linprog.py

```diff
@@ -516,11 +516,14 @@ def linprog(c, A_ub=None, b_ub=None, A_eq=None, b_eq=None,
         (c, c0, A_ub, b_ub, A_eq, b_eq, bounds, x, x0, undo, complete, status,
             message) = _presolve(c, A_ub, b_ub, A_eq, b_eq, bounds, x0, rr, tol)
 
-    postsolve_args = (c_o, A_ub_o, b_ub_o, A_eq_o, b_eq_o, bounds, undo)
+    postsolve_args = (c_o, A_ub_o, b_ub_o, A_eq_o, b_eq_o, bounds, undo, 1, 1, 1)
```

Comment:
What do the additional ``1, 1, 1`` represent? Do you think it would be neater to explicitly define them as variables above?

Reply:
done, and commented.